lib/nroonga.js

```diff
@@ -1,2 +1,6 @@
 'use strict'
+if (process.env.GROONGA_PATH) {
+  const path = require('path')
+  process.env.PATH = process.env.GROONGA_PATH + path.sep + 'bin' + path.delimiter + process.env.PATH
```

Comment:
If you use `path`, it is better to use `path.join()`.

Reply:
@abetomo Thanks, I've fixed it!